tests/benchdnn/inputs/graph/pattern/f8/f8_conv_add_add_fusion.json

```diff
@@ -134,128 +135,6 @@
         }
       ]
     }, 
-    {
```

Comment:
remove `--expected-n-partitions=0` at below position? https://github.com/uxlfoundation/oneDNN/blob/1ae1905345983d74c302ac12f8d9f9ada7022ab5/tests/benchdnn/inputs/graph/pattern/harness_f8_all#L2

Reply:
It's because fp8 fusions are not supported on old platforms before SPR and several partitions will be returned. We have to skip the partition number check for this case as it will run on all platforms.

Reply:
fp8 data-typed-cases should be reported as skipped for systems below AVX512_CORE_FP16 due to lack of emulation support.

Reply:
Currently the logic in graph driver is that, even there are multiple partitions returned, if each of them is implemented, we still test the partitions one by one. In this case, as fp8 conv is not supported before SPR, several partitions (fp8 quant/dequant, f32 conv, etc.) are returned. fp8 quant/dequant are actually supported through reorders on the platforms, so we don't skip the case.